src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/Utils.scala

```diff
@@ -55,4 +62,80 @@ object Utils {
       tenant.attributes.mapFilter(_.certified.map(_.id)) ++
       tenant.attributes.mapFilter(_.declared.map(_.id))
 
+  def isUpgradable(
```

Comment:
Does it still need to be swapped?

Reply:
what do you mean with "swapped"?
If it's about using the process instead of the management, IMO yes, we should always use the processes in the BFF, unless specific reasons (mostly for authorization)

Reply:
Yep, that's what I meant

Reply:
IMO we can manage the migration to the catalog process in a separated PR 👍